packages/docusaurus-theme-classic/src/theme/Heading/index.tsx

```diff
@@ -38,6 +38,7 @@ export default function Heading({as: As, id, ...props}: Props): JSX.Element {
       <Link
         className="hash-link"
         to={`#${id}`}
+        aria-labelledby={id}
         title={translate({
           id: 'theme.common.headingLinkTitle',
           message: 'Direct link to heading',
```

Comment:
Don't we want to update this to `Direct link to {headingTitle}`? 

I think this was what we wanted in the original issue? (this would require modifying all our translation files to add the extra param/placeholder)

Reply:
We can, and I think it would be great if we could do this:

```jsx
<Link
  aria-label={`Direct link to ${headingTitle}`} // to set the accessible name
  title={`Direct link to ${headingTitle}`} // this will be shown to users who hover over the link
/>
```
(just with that translation syntax)

Reply:
👍 let me edit the PR I'll do this